### DISCUSSION TITLE
Mac連接求助

### DISCUSSION THREAD

**Discussion by kobi1108** [MaaAssistantArknights/MaaAssistantArknights#7396] [Q&A]
Mac 版本：M2 Ventura
Maa 版本：最新
PlayCover: [v3.0.0.maa.4](https://github.com/hguandl/PlayCover/releases/tag/v3.0.0.maa.4)
方舟版本：最新脫殼的明日方舟客戶端安裝包
端口号改爲1718
LinkStart無反應
![fdafdsa](https://github.com/MaaAssistantArknights/MaaAssistantArknights/assets/140871439/20b0b3f3-faf4-4445-9c72-6622e314d6ae)
![fdsgs](https://github.com/MaaAssistantArknights/MaaAssistantArknights/assets/140871439/52f99a3d-30ba-4845-b340-b81af576c915)

**Comment by ABA2396** [comment]
cc @hguandl

**Comment by hguandl** [comment]
请尝试关闭adb-lite

**Reply by kobi1108** [reply]
![image](https://github.com/MaaAssistantArknights/MaaAssistantArknights/assets/140871439/ba0b3e97-aa4b-4f63-aa65-a99dc41d1560)
已嘗試關閉 能開啟app端 但沒辦法自動點擊

**Reply by hguandl** [reply]
可能得看一下日志了，在“文件”>“打开日志文件夹”里。

**Reply by kobi1108** [reply]
Mac 版本 找遍了maa軟件的按鈕也毫無頭緒文件夾的位置
window版本很容易就找到了qaq

**Reply by hguandl** [reply]
菜单栏的“文件”
<img width="395" alt="image" src="https://github.com/MaaAssistantArknights/MaaAssistantArknights/assets/10684225/bb4488cc-21fb-41e8-8e31-8aa399d15c01">

**Reply by kobi1108** [reply]
[asst.log](https://github.com/MaaAssistantArknights/MaaAssistantArknights/files/13533323/asst.log)
剛剛重新下載maa，play cover，host改回1717，問題依舊，以上是最新的log

**Reply by hguandl** [reply]
基建换班配置看起来有问题，无法识别，换成默认换班看看。

**Reply by kobi1108** [reply]
導入配置后成功運作了，但幹員識別好像有點問題，2K 16:9，或者其他分辨率也一樣
![螢幕擷取畫面 2023-12-03 114947](https://github.com/MaaAssistantArknights/MaaAssistantArknights/assets/140871439/9ba2d81c-99e9-49ac-9db8-d082d4d9587b)

[asst.bak.log](https://github.com/MaaAssistantArknights/MaaAssistantArknights/files/13537997/asst.bak.log)
